src/pages/civic/index.vue

```diff
@@ -33,10 +34,36 @@ export default {
         {
           hid: 'og:image',
           property: 'og:image',
-          content: 'https://liker.land/images/og/civic-v3.png',
+          content: ogImage,
         },
       ],
       link: [{ rel: 'canonical', href: `${this.$route.path}` }],
+
+      script: {
+        hid: 'schema',
+        innerHTML: JSON.stringify([
+          {
+            '@context': 'http://www.schema.org',
+            '@type': 'Product',
+            name: title,
+            image: [ogImage],
+            description,
+            brand: {
+              '@type': 'Brand',
+              name: 'Civic Liker',
+            },
+            url: this.$route.path,
```

Comment:
Need full URL?

Reply:
not sure, added in both civic and portfolio page now